admin/public/js/components/erizo_item.jsx

```diff
@@ -0,0 +1,63 @@
+define(['pubsub', 'react', 'jsx!components/erizo_publisher'], function (PubSub, React, ErizoPublisher) {
+
+  return React.createClass({
+    displayName: 'ErizoItem',
+    
+    getInitialState: function() {
+      return {
+        publisherMetadata: {}
+      };
+    },
+
+    render: function() {
+      var that = this;
+
+      var createItem = function(key) {
+        return (
+            <ErizoPublisher item={that.state.publisherMetadata[key]} key={that.props.item.id + "-" + key} pub={key} stats={that.props.stats[key]}/>
+        );
+      };
+
+      return (
+        <div className="erizo_item panel panel-primary">
+            <div className="panel-heading">
+              <div>ErizoJS_{this.props.item.id}</div>
+              <div><span className="label label-default">PID</span><span className="label label-info">{this.props.item.pid}</span></div>
+              <div><span className="label label-default">State</span><span className={"label label-" + (this.props.item.idle ? "default" : "success")}>{this.props.item.idle ? "idle" : "busy"}</span></div>
+            </div>
+            { (Object.keys(that.state.publisherMetadata).length == 0 ? "" : (
+              <div className="panel-body">
+                { Object.keys(that.state.publisherMetadata).map(createItem) }
+              </div>
+            )) }
+        </div>
+      );
+    },
+    
+    updatePublisherMetadata: function() {
+      var that = this;
+
+      PubSub.call("ErizoJS_"+ this.props.item.id + ".getPublisherMetadata", null, function(resp) {
+        if (that.isMounted()) {
+          that.setState({publisherMetadata: resp});
+        }
+      });
```

Comment:
As written, each instance of `ErizoPublisher` will set its own timer and call the `PubSub` endpoint every two seconds; if you have an endpoint that can return responses for *multiple* items at once, you could consider moving the data fetching to a higher-level component using a single timer, and then pass that data into the necessary subcomponents.

This is also a good use case for outside-of-React data storage patterns, like flux — some *other* system periodically updates some external data, and then React re-renders things any time that data changes (sort of like a centralized `setState`).